tests/es6-export.js

```diff
@@ -122,7 +122,7 @@ describe("export declaration", () => {
     });
 
     it("should refer exported references#1", () => {
-        const ast = espree("export {x};");
+        const ast = espree("const x = 1; export {x};");
```

Comment:
More recent versions of Espree will throw an error if the export is not defined: `SyntaxError: Export 'x' is not defined`